includes/class-wc-install.php

```diff
@@ -628,6 +626,24 @@ private static function get_schema() {
   PRIMARY KEY (log_id),
   KEY level (level)
 ) $collate;
+CREATE TABLE {$wpdb->prefix}woocommerce_webhooks (
```

Comment:
Should we start using wc_ prefix (https://github.com/woocommerce/woocommerce/pull/16390)